api/database/seeds/ArticleSeeder.php

```diff
@@ -95,6 +95,9 @@ public function run()
                         }
                     });
 
+                $article->bookmark()->save(new \Spira\Bookmark\Model\Bookmark(['user_id' => $users->random()->user_id]));
+                $article->rate()->save(factory(\Spira\Rate\Model\Rating::class)->make(['user_id' => $users->random()->user_id]));
```

Comment:
Seed with many bookmarks and ratings (random number between 0 and user count), as we will use this later to determine popularity rankings, and if there is only one per article it will be hard to tell.